core/src/main/scala/akka/kafka/internal/SubSourceLogic.scala

```diff
@@ -115,17 +116,23 @@ private[kafka] abstract class SubSourceLogic[K, V, Msg](
               .recover {
                 case _: AskTimeoutException =>
                   stageFailCB.invoke(
-                    new ConsumerFailed(s"Consumer failed during seek for partitions: ${partitions.mkString(", ")}.")
+                    new ConsumerFailed(
+                      s"#$actorNumber Consumer failed during seek for partitions: ${partitions.mkString(", ")}."
+                    )
                   )
               }
         }
     }
   }
 
   val partitionRevokedCB = getAsyncCallback[Set[TopicPartition]] { tps =>
+    // TODO this called in startup with empty tps, some existing tests reply on the callback
     pendingRevokeCall.map(_.cancel())
     partitionsToRevoke ++= tps
     val cb = getAsyncCallback[Unit] { _ =>
+      if (log.isDebugEnabled) {
```

Comment:
I thought that `log.debug` takes arguments after format by name, thus not calling `toString` on them if `isDebugEnabled` is false. But I checked the source code and that is not the case. :/